nb/projects/plotter/mount.nb

```diff
@@ -32,7 +32,7 @@ const upsideClip = Box(30 + 4)
       .z(10.5, 0)
       .x(23)
       .hull()
-      .op((f) => s.cut(f.extrudeAlong(normal(), -15)).and(f.ex(0.4)))
+      .op((f) => s.cut(f.extrudeAlong(normal(), -15)).and(f.extrudeAlong(normal(), 0.4)))
```

Comment:
This is neat! I remember talking about the idea, but I didn't realize it was already working.